package.json

```diff
@@ -2,34 +2,47 @@
   "name": "webhook-handler",
   "version": "1.0.0",
   "scripts": {
-    "build": "ENVIRONMENT=local npm run build:config && sls webpack",
-    "build:config": "tsc src/config.ts",
-    "clean": "skripts clean scripts --pattern '*.js'",
-    "deploy": "npm run build:config && sls deploy && npm run update:all",
-    "format": "skripts format",
-    "jest": "PARTNER_QUEUE_URL=partner.com RESULT_QUEUE_URL=result.com ERROR_QUEUE_URL=error.com VERSION=v1 jest",
-    "lint": "skripts lint --fix",
+    "build": "export ENVIRONMENT=local && npm run build:config && sls webpack",
+    "build:config": "tsc --project ./tsconfig.json",
+    "deploy": "npm run build:config && sls deploy",
```

Comment:
Its worth calling out the `npm run update:all` has been removed from the deploy process. This was in charge of updating existing clients webhook handlers code. We want to have more control and slowly roll this out. 

That being said if a client has not created a webhook subscription before they will get the newer version of the code.

Reply:
What is the end goal once the upgrade is complete? Would this be reintroduced?

Reply:
We have some follow up tickets to figure out the best strategy for this. We might reintroduce it or have a new lambda that allows a slower roll out before running the update all command